queries/wiki.py

```diff
@@ -124,17 +125,39 @@ def help_text(lemma):
     | "gætirðu" "flett" "upp" QWikiSubjectÞgf "í" QWikipedia
 
 QWikiSubjectNf →
+    QWikiPrevSubjectNf | QWikiSubjectNlNom
```

Comment:
Better to not mix up Icelandic Nf/Þf/Þgf/Ef with English Nom/Acc/Dat/Gen?

Reply:
So true. I'm actually a bit confused about this and I think we should establish strict conventions. So we stick with the Icelandic names of cases? So it's nf, þf, þgf and ef, as opposed to nom, acc, dat, gen?

Reply:
When you think about it, this probably makes the codebase more transparent for Icelandic speakers at the cost of making it more opaque for others. Decisions, decisions!